compiler/modules/front.syntax/ast.bal

```diff
@@ -107,7 +107,7 @@ public type IfElseStmt record {|
     *PositionFields;
     Expr condition;
     StmtBlock ifTrue;
-    StmtBlock? ifFalse;
+    StmtBlock ifFalse;
```

Comment:
Why are you doing this?  The way it was is fine. You just need to fix codeGen.bal.

Reply:
fixed